http-clients/aws-crt-client/src/main/java/software/amazon/awssdk/http/crt/AwsCrtAsyncHttpClient.java

```diff
@@ -417,6 +437,10 @@ public void close() {
         private EventLoopGroup eventLoopGroup;
         private HostResolver hostResolver;
         private ProxyConfiguration proxyConfiguration;
+        private HttpMonitoringOptions monitoringOptions;
+
+        // default reaping interval matches Apache client
+        private Duration connectionMaxIdleTime = Duration.ofSeconds(60);
```

Comment:
Minor: I don't think we need to create a default `connectionMaxIdleTime` here since the default value will be merged in when we merge `SdkHttpConfigurationOption.GLOBAL_HTTP_DEFAULTS` in the build method.

```
        @Override
        public SdkAsyncHttpClient build() {
            return new AwsCrtAsyncHttpClient(this, standardOptions.build()
                                                                  .merge(SdkHttpConfigurationOption.GLOBAL_HTTP_DEFAULTS));
        }
```